packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -668,6 +679,60 @@ class Auth {
         return await this.loginGuestUser()
     }
 
+    /**
+     * A wrapper method for commerce-sdk-isomorphic helper: authorizeIDP.
+     *
+     */
+    async authorizeIDP(parameters: Parameters<Helpers['authorizeIDP']>[1]) {
+        const privateClient = !!this.clientSecret
+        const redirectURI = this.redirectURI
+        const usid = this.get('usid')
+        const {url, codeVerifier} = await helpers.authorizeIDP(
+            this.client,
+            {
+                redirectURI,
+                hint: parameters.hint,
+                ...(usid && {usid})
+            },
+            privateClient
+        )
+        window.location.assign(url)
+        this.set('code_verifier', codeVerifier)
+    }
+
+    /**
+     * A wrapper method for commerce-sdk-isomorphic helper: loginIDPUser.
+     *
+     */
+    async loginIDPUser(parameters: Parameters<Helpers['loginIDPUser']>[2]) {
+        const codeVerifier = this.get('code_verifier')
+        const code = parameters.code
+        const usid = parameters.usid
+        const redirectURI = parameters.redirectURI || this.redirectURI
```

Comment:
Will this helper throw at all if any of these values aren't passed in?

Reply:
When I was testing in the `retail-react-app`, the helpers didn't throw if I didn't pass in `code`, a required param, but the API call would respond with a `401` "invalid authorization code" error